baseplate/integration/pyramid/__init__.py

```diff
@@ -168,6 +168,10 @@ def __init__(self, baseplate, trust_trace_headers=None,
                 trust_headers=self.trust_trace_headers,
             )
 
+    def _on_application_created(self, event):
+        # attach the baseplate object to the application the server gets
+        event.app.baseplate = self.baseplate
```

Comment:
🔕 what's going on here?

Reply:
making the `Baseplate` object accessible to the server in a standard location across thrift/wsgi server types.

Reply:
ah, so we can do https://github.com/reddit/baseplate/pull/242/files#diff-ff6fa7da3dc7ab489d21610e7e8b6527R37 ?

Reply:
yup!